.github/workflows/build.yml

```diff
@@ -389,11 +391,13 @@ jobs:
     needs:
       - set-product-version
       - product-metadata
-      - build-linux
+      - build-docker
```

Comment:
- [ ] Revert before merge (will change this once we have the enos scenarios in place)